content/en/_index.md

```diff
@@ -218,6 +302,18 @@ a file or a specific action is not.
         // Snippet content ...
     }
   {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    function main(requiredArg) {
```

Comment:
nit: I think this is supposed to have no args
```suggestion
    function main() {
```

Reply:
Done.